examples/nextjs-swc/src/components/Developers.tsx

```diff
@@ -17,10 +17,7 @@ export default function Developers() {
           <option value={'2'}>2</option>
         </select>
         <p>
-          {plural(selected, {
-            one: 'Developer',
-            other: `Developers`
-          })}
+          <Plural value={selected} one={'Developer'} other={`Developers`} />
```

Comment:
@yunsii whenever non-react macro (t, plural, select, etc) is used in react component, `useLingui()` hook should be added as well. OR just use JSX macro instead. 

That caused a translation mismatch you noticed in the issue. 

Here i've just changed it to JSX macro, because it's more appropriate usage.

Reply:
It means if i don't use i18n, should still call useLingui hook like below?

```tsx
import { useLingui } from "@lingui/react"
import { plural } from '@lingui/macro'

function App() {
  useLingui()
  //...

  return (
    <p>
      {plural(selected, {
        one: 'Developer',
        other: `Developers`
      })}
    </p>
  )
}
```

Reply:
yes. plural / t / select / selectOrdinal macro is simply a syntax sugar over i18n, it is expanded to the `i18n._` call during build.

I recommend to use react-related macros whenever possible, because it auto subscribe for a changes and less error-prone.

Reply:
Theaks for your detail explanation 👌